From e8bb90e42d0356cdcf91d22c9deb6635d1d3c376 Mon Sep 17 00:00:00 2001
From: Matej Habrnal <mhabrnal@redhat.com>
Date: Mon, 2 Feb 2015 21:41:36 +0100
Subject: [PATCH 1004/1015] conf: changed URL for sending uReport

Changed faf server url in report_uReport.xml.in.
uReports are sending to https://retrace.fedoraproject.org/faf by default.

Signed-off-by: Matej Habrnal <mhabrnal@redhat.com>
---
 src/plugins/report_uReport.xml.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/plugins/report_uReport.xml.in b/src/plugins/report_uReport.xml.in
index b997851..eca30e0 100644
--- a/src/plugins/report_uReport.xml.in
+++ b/src/plugins/report_uReport.xml.in
@@ -12,7 +12,7 @@
             <_label>uReport Server URL</_label>
             <allow-empty>no</allow-empty>
             <_description>Address of uReport webservice</_description>
-            <default-value>http://bug-report.itos.redhat.com</default-value>
+            <default-value>https://retrace.fedoraproject.org/faf</default-value>
         </option>
         <option type="text" name="uReport_ContactEmail">
             <_label>Contact email address</_label>
-- 
1.8.3.1

